indexer/feature_meta.cpp

```diff
@@ -50,14 +36,33 @@ std::string Metadata::GetWikiURL() const
   return ToWikiURL(string(Get(FMD_WIKIPEDIA)));
 }
 
-string Metadata::ToWikimediaCommonsURL(std::string const & v)
+std::string Metadata::ToWikimediaCommonsURL(std::string v)
 {
   if (v.empty())
     return v;
 
+  v = EncodeWikiURL(v, 0);
+
   return "https://commons.wikimedia.org/wiki/" + v;
 }
 
+std::string Metadata::EncodeWikiURL(std::string v, int startIndex) {
```

Comment:
The usage pattern suggests to avoid unnecessary copying:
```suggestion
void Metadata::EncodeWikiURL(int startIndex, std::string & url)
{
```